pages/index.md

```diff
@@ -6,38 +6,97 @@ PyCon UK is the annual gathering of the UK Python community and its friends from
 This year, we return to [Cardiff City Hall](http://www.cardiffcityhall.com/),
 from Thursday 26th to Monday 30th October.
 
-* * *
+**_Sold Out_**: we are completely sold out with over 700 people coming. We regret
+that we have no waiting list. If you are a sponsor or a speaker with a query about
+tickets please contact the [organising committee](/contact/)    .
 
-Tickets for the main conference have now sold out, and we're expecting over 700 people at the conference!
-(But there are still a handful of tickets left for the [children's day](https://hq.pyconuk.org/children/orders/new/).)
+We [still have tickets](https://hq.pyconuk.org/children/orders/new/) for the 
```

Comment:
“still have *a handful of*”?

And is this still true?

Reply:
Yes.

Reply:
Yes.

Reply:
Just a bit more compact with a lot else on the page. Yes; Carrie Anne says they can take 100 -- we're at 77